.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: java
-dist: trusty
```

Comment:
I would guess it's running on different machines, not so much related to the distribution per se.
Some time ago I noticed builds on precise got about twice faster, with no configuration changes.